n3fit/src/evolven3fit_new/eko_utils.py

```diff
@@ -71,7 +94,7 @@ def construct_eko_for_fit(theory_card, op_card, save_path=None):
             raise FileNotFoundError(
                 f"Path where eko should be dumped does not exist: {save_path}"
             )
-    eko_op = run_dglap(theory_card, op_card)
+    eko_op = runner.solve(theory_card, op_card, save_path)
```

Comment:
One question, if the `runner` is already receiving `save_path` we no longer need the lines below, right? @andreab1997

Reply:
Yes I'd say so.

Reply:
Yes we can remove those lines